tests/src/integration/runtimetests/manifest.yml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: helloworld
+  version: 1.0
+  license: Apache-2.0
+  actions:
+    greeting:
+      web-export: true
+      version: 1.0
+      function: src/greeting.js
+      runtime: nodejs:6
+      inputs:
+        name: string
+        place: string
+      outputs:
+        payload: string
+  actions:
+      greetingphp:
+        web-export: true
+        version: 1.0
+        function: src/hello.php
+        runtime: php:7.1
+        inputs:
+          name: string
+          place: string
+        outputs:
+          payload: string
+  actions:
+        greetingpython:
+          web-export: true
+          version: 1.0
+          function: src/hello.py
+          runtime: php:7.1
+          inputs:
+            name: string
+            place: string
+          outputs:
+            payload: string
+  actions:
+          greetingswift:
+            web-export: true
+            version: 1.0
+            function: src/hello.swift
+            runtime: php:7.1
+            inputs:
+              name: string
+              place: string
+            outputs:
+              payload: string
+  actions:
+          helloworldjava:
+            function: src/hello.jar
+            runtime: java
+            main: Hello
```

Comment:
@lzbj Should we test an invalid value to test our response? 

Also, I would like to see a valid "kind" (like NodeJS) used with an old (no longer supported) version to test for that as well.

Reply:
ok, I'll add more test.